packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -275,6 +278,17 @@ class RadioListTile<T> extends StatelessWidget {
   /// If non-null, defines the background color when [RadioListTile.selected] is true.
   final Color? selectedTileColor;
 
+  /// Defines how compact the list tile's layout will be.
+  ///
+  /// {@macro flutter.material.themedata.visualDensity}
+  final VisualDensity? visualDensity;
+
+  /// {@macro flutter.widgets.Focus.focusNode}
+  final FocusNode? focusNode;
+
+  /// {@macro flutter.material.ListTile.enableFeedback}
+  final bool? enableFeedback;
```

Comment:
Can you also add the "see also" section here, if you think it's useful (and in `SwitchListTile`:

```
  /// See also:
  ///
  ///  * [Feedback] for providing platform-specific feedback to certain 
```

Reply:
Yep, sure, I've added this